juan_vides/ejercicio.md

```diff
@@ -0,0 +1,31 @@
+``` 
+Sub ejercicio()
+    abo = 0
+    noabo = 0
+    abosup = 0
+    Total = 0
+    
+    While Total <= 3000000
+        dinero = Int(InputBox("Cuanto va a abonar?"))
+        If dinero > 0 Then
+            abo = abo + 1
+            Total = Total + dinero
```

Comment:
The same recommendation that I made to a colleague, for the variables it is more recommended that everything be in lowercase